nilearn/datasets/func.py

```diff
@@ -98,6 +98,14 @@ def fetch_haxby(data_dir=None, n_subjects=1, fetch_stimuli=False,
     n_subjects: int, optional
         Number of subjects, from 1 to 6.
 
+        NOTE: n_subjects is deprecated from 0.2.5.1 and will be removed in 0.3
+        Use `subjects` instead.
+
+    subjects : int or list, optional
```

Comment:
+1
Also, in the few fetchers that allows subject selection, we usually use subject_ids or sub_ids.